src/helpers/randomizeCard/index.js

```diff
@@ -0,0 +1,736 @@
+			"Feathered",
+			"Flocking",
+			"Corvid",
+			"Soulless",
+			"Occult",
+			"Plumed"
+		],
+		[
+			" Hags",
+			" Prophets",
+			" Harpies",
+			" Deceivers",
+			" Witches",
+			" Shamans",
+			" Ministers",
+			" Cultists",
+			" Stalkers",
+			" Augurs",
+			" Conjurers",
+			" Harbringers",
+			" Omens"
+		]
+	],
+	["toad",
+		[
+			"Brood",
+			"Copperskin",
+			"Crimson",
+			"Lime",
+			"Azure",
+			"Obsidian",
+			"Plagued",
+			"Hairy",
+			"Salty",
+			"Vermilion",
+			"Viridian",
+			"Scarlet",
+			"Malachite",
+			"Amber",
+			"Rusthide"
+		],
+		[
+			" Sages",
+			" Rangers",
+			" Sentries",
+			" Troopers",
+			" Hatchers",
+			" Butchers",
+			" Chestnuts",
+			" Tadpoles",
+			" Polliwogs",
+			" Sycophants",
+			" Amphibians",
+			" Leapers"
+		]
+	],
+	["frostling",
+		[
+			"Frosty",
+			"Fel",
+			"Orgone",
+			"Wisp",
+			"Blizzard",
+			"Chilled",
+			"Dawn",
+			"Iced",
+			"Calming"
+		],
+		[
+			" Hexers",
+			" Flakes",
+			" Flares",
+			" Leechers",
+			" Clouds",
+			" Bombs",
+			" Stonemanes",
+			" Sparks",
+			" Droplings",
+			" Visions",
+			" Spirits",
+			" Channelers"
+		]
+	],
+	["dwarf",
+		["Myst","Snow","Rock","Hearth","Wolf","Earth","Sleet","Chill","Hail","Frost","Ore","Shale","Stone"],
+		["wives","masons","workers","guards","cloaks","fathers","menders","stompers","beards","smiths","forgers","singers"]
+	],
+	["rodent",
+		["Ozone","Sound","Absorbing","Windy","Chaotic","Armed","Booming","Mechanical","Crazy"],
+		[" Lackeys"," Purifiers"," Drivers"," Varmints"," Minions"," Launchers"," Chargers"," Agents"," Pupils"," Schemers"," Professors"," Officers"," Workers"," Bombers"]
+	],
+	["construct",
+		["Function","Linked","Finite","Scrapped","Debug","Projected","Plated","Oxidized","Sleek","Automated","Hardwired","Boolean","Software","Motorized","Analog"],
+		[" Prototypes"," Wilds"," Golems"," Loopers"," Servers"," Planners"," Loggers"," Automatons"," Apparati"," Engines"]
+	],
+	["satyr",
+		["Lawless","Restless","Mindless","Pan","Wasteland","Desolate","Arid","Collective","Horned"],
+		[" Herd"," Goats"," Companions"," Horde"," Shepards"," Heralds"," Bucks"," Fawns"]
+	],
+	["undead",
+		["Forgotten","Shady","Mischevious","Petrified","Grim","Vindicative","Lasting"],
+		[" Regrets"," Souls"," Ghouls"," Liches"," Summoners"," Fossils"," Couriers"," Harvesters"," Martyrs"," Remains"]
+	],
+	["pirate",
+		["Northsea","Westwind","Cabin","Bluesail","Lucky","Seasick","Starboard"],
+		[" Dogs"," Mutineers"," Looters"," Sailors"," Privaters"," Captains"," Raiders"," Swindlers"," Charmers"," Bouncers"]
+	]
+]
+
+var statlines = [
+	[2,4,5,7,8,10,11,13], // 0 Speed
+	[1,2,3,5,6,8,9,11],	 // 1 Speed
+	[0,1,2,3,4,5,6,7],		// 2 Speed
+	[0,0,1,2,3,4,5,6]		 // 3 Speed
+]
+
+//###########################
+//# Code to generate a card #
+//###########################
+
+class Card {
+	constructor() {
+		this.mana = 0
+		this.effCost = 0
+		this.effCostMult = 0
+		this.str = 0
+		this.mov = 0
+		this.tribe = ""
```

Comment:
To align with the rest of the site: `faction`. Or `race`? Not sure what you count as a tribe.